exercises/concept/amusement-park/amusement-park.js

```diff
@@ -0,0 +1,67 @@
+// @ts-check
+
+/**
+ * Creates a new visitor.
+ *
+ * @param {string} name
+ * @param {number} age
+ * @param {string} ticketId
+ * @returns {Object} visitor
```

Comment:
It might be worth using actual "types" for the visitors. For example you can define `global.d.ts` with:

```typescript
type Visitor= {
  name: string
  age: number
  ticketId: string | null
}
```

And then you can use `@returns {Visitor} the object with visitor data`

Reply:
I looked at another exercise where you did this and I think I got it sorted as you suggested.